### PR TITLE
Upgrade ember-power-select-blockless and pass all options.

### PR DESCRIPTION
**Pull request by nbrookie**
Pass all possible options down to the ember-power-select/ember-power-select-blockless components, as well as take care of https://github.com/kaliber5/ember-bootstrap-power-select/pull/30

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "ember-load-initializers": "^1.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-power-select": "^2.0.0",
-    "ember-power-select-blockless": "^0.3.1",
+    "ember-power-select-blockless": "0.5.0",
```
Comment (simonihmig) [inline package.json]: Can you change that to `^0.5.0`, so it can pick up new patch releases!

Comment (simonihmig) [reply package.json]: Forget it, just remembered I can do this by my own in Github... :)

**Comment by seansellek** [comment]
Can we cut a new release to take advantage of this change? It adds a lot of important options for eps

**Comment by simonihmig** [comment]
@seansellek thanks for the reminder! Just published 1.0.0! 🎉

**Pull request merged**